src/plugins/intel_cpu/src/node.cpp

```diff
@@ -170,10 +181,20 @@ Node::Node(const std::shared_ptr<ngraph::Node>& op, const dnnl::engine& eng, Wei
     }
 }
 
-Node::Node(const std::string& type, const std::string& name, const dnnl::engine& eng, WeightsSharing::Ptr &w_cache)
-        : selectedPrimitiveDescriptorIndex(-1), permanent(false), temporary(false), constant(ConstantType::Unknown),
-          weightCache(w_cache), engine(eng), fusingPort(-1), name(name), typeStr(type),
-          type(TypeFromName(type)), profiling(name) {
+Node::Node(const std::string& type, const std::string& name, GraphContext::Ptr ctx)
+    : selectedPrimitiveDescriptorIndex(-1),
+      permanent(false),
+      temporary(false),
+      constant(ConstantType::Unknown),
+      context(ctx),
+      engine(ctx->getEngine()),
+      weightCache(ctx->getWeightsCache()),
+      isInQuantizedGraph(ctx->getGraphQuantizedFlag()),
```

Comment:
Could you please albacore a little bit on why we should keep `weightCache` and `isInQuantizedGraph` in the node, while `rtParamsCache`, `isInQuantizedGraph` and `rtScratchPad` were removed, and all these data members can be read from the context?

Reply:
I was lazy, sorry, fixed